addons/containerd/template/base/install.sh

```diff
@@ -193,7 +193,7 @@ function containerd_configure_proxy() {
         return
     fi
 
-    log "Updating proxy configuration with new changes"
+    log "Updating proxy configuration: HTTP_PROXY=${PROXY_ADDRESS} NO_PROXY=${NO_PROXY_ADDRESSES}"
```

Comment:
I was hoping you could make this change across all files.

Reply:
Yes it is done/